packages/teleterm/src/ui/DocumentCluster/ClusterResources/Kubes/Kubes.tsx

```diff
@@ -80,3 +113,14 @@ export const renderConnectButtonCell = (
     </Cell>
   );
 };
+
+const DarkenWhileDisabled = styled(Box)`
+      // The timing functions of transitions have been chosen so that the element loses opacity slowly
+      // when entering the disabled state but gains it quickly when going out of the disabled state.
+      transition: opacity 150ms ease-out;
+      &.disabled {
+        pointer - events: none;
```

Comment:
It should be replaced by a common `DarkenWhileDisabled` anyway, but something weird happened in this line (I mean `pointer - events`).